packages/interactivity-router/src/index.js

```diff
@@ -90,38 +100,56 @@ export const { state, actions } = store( 'core/router', {
 		 * needed, and updates any interactive regions whose contents have
 		 * changed. It also creates a new entry in the browser session history.
 		 *
-		 * @param {string}  href              The page href.
-		 * @param {Object}  [options]         Options object.
-		 * @param {boolean} [options.force]   If true, it forces re-fetching the
-		 *                                    URL.
-		 * @param {string}  [options.html]    HTML string to be used instead of
-		 *                                    fetching the requested URL.
-		 * @param {boolean} [options.replace] If true, it replaces the current
-		 *                                    entry in the browser session
-		 *                                    history.
-		 * @param {number}  [options.timeout] Time until the navigation is
-		 *                                    aborted, in milliseconds. Default
-		 *                                    is 10000.
+		 * @param {string}  href                           The page href.
+		 * @param {Object}  [options]                      Options object.
+		 * @param {boolean} [options.force]                If true, it forces re-fetching the URL.
+		 * @param {string}  [options.html]                 HTML string to be used instead of fetching the requested URL.
+		 * @param {boolean} [options.replace]              If true, it replaces the current entry in the browser session history.
+		 * @param {number}  [options.timeout]              Time until the navigation is aborted, in milliseconds. Default is 10000.
+		 * @param {boolean} [options.loadingAnimation]     Whether an animation should be shown while navigating. Default to `true`.
+		 * @param {boolean} [options.screenReaderAnnounce] Whether a message for screen readers should be announced while navigating. Default to `true`.
 		 *
-		 * @return {Promise} Promise that resolves once the navigation is
-		 *                   completed or aborted.
+		 * @return {Promise} Promise that resolves once the navigation is completed or aborted.
 		 */
 		*navigate( href, options = {} ) {
-			const url = cleanUrl( href );
+			const pagePath = getPagePath( href );
+			const { navigation } = state;
+			const {
+				loadingAnimation = true,
+				screenReaderAnnounce = true,
```

Comment:
Shouldn't this be `screenReaderAnnouncement`?

Reply:
Oops! Fixed in https://github.com/WordPress/gutenberg/pull/58377/commits/e08f981a36ccae4b338ed79273c6876e941e5533